npm/webpack-dev-server/src/startServer.ts

```diff
@@ -2,17 +2,23 @@ import webpack from 'webpack'
 import { debug as debugFn } from 'debug'
 import WebpackDevServer from 'webpack-dev-server'
 import fw from 'find-webpack'
+
 const debug = debugFn('cypress:webpack-dev-server:start')
 
 import { makeWebpackConfig } from './makeWebpackConfig'
 
-export async function start (userWebpackConfig = {}, { specs, config, devserverEvents }): Promise<WebpackDevServer> {
-  const foundConfig = fw.getWebpackOptions()
-  const webpackConfig = await makeWebpackConfig(foundConfig || userWebpackConfig, {
+export async function start (initialWebpackConfig, { specs, config, devServerEvents }): Promise<WebpackDevServer> {
+  if (initialWebpackConfig == null) {
+    debug('User did not pass in any webpack configuration')
+    // TODO: implement a method in find-webpack to return the path where it found the configuration
+    initialWebpackConfig = fw.getWebpackOptions()
+  }
+
+  const webpackConfig = await makeWebpackConfig(initialWebpackConfig, {
```

Comment:
Bug fix, we stopped preferring the user's configuration and instead always used findWebpack 😬